### ISSUE TITLE
Reproducing MARTHE's learning rate schedules

### ISSUE THREAD

**Issue by Gyeongjo** [awslabs/adatune#2]
Hi, I am trying to reproduce the interesting results presented in figure 3 of your paper, especially the learning rate schedules your algorithm found.
However, a few trials with different `alpha` and seeds on CIFAR10 give me LR schedules that look like exponential decay (i.e. no increase in LR in the early stages of training).
Here is the graph of LR schedules that I found:
<img width="839" alt="learning_rates" src="https://user-images.githubusercontent.com/20377824/70736395-9525e080-1d53-11ea-9529-43503c1546ac.png">

I assume that the example schedule presented in figure 3 is a special case (for some specific seed) or I am missing something. Would you like to share your ideas on this?

(Legend string MARTHE_(1)_(2)_(3) represents (1): 1st or 2nd order, (2): alpha, (3): seed where all the other hyperparameters are set to the values as described in the paper.)

Thanks in advance!

**Comment by jmikko** [comment]
(following our emails I am answering also here)

Hi @GyeongJoHwang 
thank you again for your interest in our project!

What you are saying is right, there is no constraint in the shape of the LR schedule for MARTHE, and in this sense only some initializations\minibatch orders\etc… will have that particular shape. We have to consider that even the randomization in the GPU can change the resulting scheduling of the LR.

In fact, the goal of our method (and the adaptive methods in general) is to avoiding the burden of worrying about the shape of the LR itself, and the adaptation takes care of that for you.

Thank you again,
Michele.

Side note: as a sanity check, you could try to use an initial LR very small (very close to 0.0).
In this very disadvantage scenario, you should note an increase before the rapid decrease.

**Comment by Gyeongjo** [comment]
Hi @jmikko 
Thanks for the kind reply :)

Your explanation clearly answers the questions that I raised.

I will close the issue.
Gyeongjo
